3741/3/helpers/review_helper.rb

```diff
@@ -0,0 +1,16 @@
+# frozen_string_literal: true
+
+require 'sinatra/base'
+
+module ReviewHelper
```

Comment:
Methods described here is just a private methods of controller

Reply:
yes, same reason as above